### ISSUE TITLE
Threshold should only consider female lice per fish

### ISSUE THREAD

**Issue by erolm-a** [resistance-modelling/slim#131]
From [this document](https://www.salmon-trout.org/wp-content/uploads/2017/08/STC-Scotland-2013-2015-The-Control-of-Sea-Lice-on-Fish-Farms-in-Scotland.pdf)

> Both the NTS and the Code of Good Practice suggest treatment thresholds 
for the application of medicines to achieve this. The suggested thresholds are 0.5 
**adult female lice** (Lepeoptherius salmonis) per fish _between February and June_ and 
1.0 adult female lice (L. salmonis) per fish _between July and January_”

In other words the ratios are specific to AFs and time-dependent. The second constraint may be relaxed for now but the first one is to be addressed first.
